pkg/lib/oauth/handler/handler_anonymous_user.go

```diff
@@ -228,7 +228,12 @@ func (h *AnonymousUserHandler) signupAnonymousUserWithRefreshTokenSessionType(
 		return nil, err
 	}
 
-	dpopJKT, _ := dpop.GetDPoPProofJKT(ctx)
+	dpopJKT, _, err := dpop.GetDPoPProofJKT(ctx)
```

Comment:
Make `dpop.GetDPoPProofJKT` take client, such that we are confident that `client.DPoPDisabled` is always respected.